bakerydemo/templates/locations/location_page.html

```diff
@@ -5,106 +5,106 @@
+                                {% if hours.closed == True %}
+                                    Closed
+                                {% else %}
+                                {% if hours.opening_time %}
+                                    {{ hours.opening_time }}
+                                {% endif %} -
+                                {% if hours.closing_time %}
+                                    {{ hours.closing_time }}
+                                {% endif %}
+                                {% endif %}
+                                </span></time>
+                        {% endfor %}
+                        </div>
+                    {% endif %}
+                    </div>
                 </div>
-            {% endif %}
-          </div>
+            </div>
         </div>
-    </div>
-</div>
-<div class="container-flex">
-  <div class="row">
-    <div class="col-md-2 col-md-offset-5 location-address">
-      <h3>Address</h3>
-        <address>{{ page.address|linebreaks }}</address>
-    </div>
-  </div>
+        <div class="container-flex">
+            <div class="row">
+                <div class="col-md-2 col-md-offset-5 location-address">
+                    <h3>Address</h3>
+                    <address>{{ page.address|linebreaks }}</address>
+                </div>
+            </div>
 
-  <div class="map-container">
-    <div id="map" class="maps embed-container"></div>
-  </div>
-</div>
+            <div class="map-container">
+                <div id="map" class="maps embed-container"></div>
+            </div>
+        </div>
 
-<div class="container">
-  <div class="row">
-    <div class="col-md-7 location-body">
-      {{ page.body }}
+        <div class="container">
+            <div class="row">
+                <div class="col-md-7 location-body">
+                    {{ page.body }}
+                </div>
+            </div>
+        </div>
     </div>
-  </div>
-</div>
 
     <script>
-      var map;
-      function initMap() {
+        var map;
+        function initMap() {
         map = new google.maps.Map(document.getElementById('map'), {
-          center: {
-              lat: {{lat}},
-              lng: {{long}}
-          },
-          zoom: 15,
-          scrollwheel: false
+            center: {
+                lat: {{lat}},
+                lng: {{long}}
+            },
+            zoom: 15,
+            scrollwheel: false
         });
         var marker = new google.maps.Marker({
-          position: {
-              lat: {{lat}},
-              lng: {{long}}
-          },
-          map: map,
-          title: '{{page.title}}'
+            position: {
+                lat: {{lat}},
+                lng: {{long}}
+            },
+            map: map,
+            title: '{{page.title}}'
         });
-      }
+        }
     </script>
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyD31CT9P9KxvNUJOwDq2kcFEIG8ADgaFgw&callback=initMap" async defer></script>
```

Comment:
Is there any real benefit of putting this in a setting variable? It's pretty tightly coupled to this piece of functionality and I can't imagine we'd want to insist that users create their own map credentials. In any case, will raise it as an issue so we can talk more about it there.